.gitignore

```diff
@@ -5,5 +5,7 @@
 
 /output.*
 
+/.idea
```

Comment:
Osobně zastávám názor, že v projektu nemají být úpravy specifické pro různá IDE. Kdyby teď někdo přišel, že používá Netbeans, tak tam bude chtít i svoje. A když Netbeansy umřou, tak nám to tam zůstane. Proto mám toto vyignorované v globálním .gitignore na svém počítači.

```
$ git config -l
core.excludesfile=~/.gitignore
$ cat ~/.gitignore 
.idea/
composer.phar
```

```
git config --global core.excludesfile ~/.gitignore
```